### PR TITLE
small generation.py doc fix

### PR DESCRIPTION
**Pull request by h-mayorquin**
As discussed in today's meeting.

### CODE REVIEW

File: src/spikeinterface/core/generate.py [hunk 1]
```diff
@@ -343,7 +346,7 @@ def synthesize_random_firings(
         The firing rate of each unit (in Hz).
         If float, all units will have the same firing rate.
     add_shift_shuffle: bool, default: False
-        Optionaly add a small shuffle on half spike to autocorrelogram
+        Optionaly add a small shuffle on half of the spikes to make the autocorrelogram less flat.
```
Comment (zm711) [inline src/spikeinterface/core/generate.py:349]:
```suggestion
        Optionally add a small shuffle on half of the spikes to make the autocorrelogram less flat.
```
while you're updating :)

**Review by zm711 — COMMENTED** [review]
One typo that can fixed since you're updating.

File: src/spikeinterface/core/generate.py (hunk 1 above)
Comment (h-mayorquin) [reply src/spikeinterface/core/generate.py]: Totally escaped us! And it was the three of us x D

File: src/spikeinterface/core/generate.py [hunk 2]
```diff
@@ -186,6 +188,7 @@ def generate_sorting(
         spikes_in_seg["segment_index"] = segment_index
         spikes.append(spikes_in_seg)
 
+        #
```
Comment (alejoe91) [inline src/spikeinterface/core/generate.py:191]:
```suggestion
```

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**